src/templates/admin/elements/layout/key_value_above.html

```diff
@@ -0,0 +1,18 @@
+<div class="key-value-pair key-above">
+  <div class="key">
+    {{ key }}
+  </div>
```

Comment:
Any way to make the label semantic relative to the value? I don't have a good way to do this in mind. I don't have a preferred way to do this, but some possibilities include using a description list, making it an `h3` (with an option for the caller to specify the h level?), or making it a `strong`.